### PR TITLE
Issue #3164390 by chr.fritsch, alexpott: Add schema for thunder_media image effect

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: modules/thunder_media/config/schema/thunder_media.schema.yml [hunk 1]
```diff
@@ -7,3 +7,17 @@ thunder_media.settings:
...
+image.effect.thunder_media_auto_aspect:
+  type: mapping
+  label: 'Auto Aspect'
+  mapping:
+    landscape:
+      type: string
+      label: 'Landscape image style'
+    portrait:
+      type: string
+      label: 'Portrait'
```
Comment (alexpott) [inline modules/thunder_media/config/schema/thunder_media.schema.yml]: `Portrait image style` as this is what it is.

**Review by alexpott — DISMISSED** [review]
We should also create a follow issue to add a \Drupal\Component\Plugin\DependentPluginInterface::calculateDependencies() implementation to \Drupal\thunder_media\Plugin\ImageEffect\AutoAspectEffect since this should be adding dependencies on the two selected image styles.

File: modules/thunder_media/config/schema/thunder_media.schema.yml (hunk 1 above)
Comment (alexpott) [reply modules/thunder_media/config/schema/thunder_media.schema.yml]: Another thought - is this translatable? I can conceive of a use case where the image style is adding copyright text. But maybe let's leave that can-of-worms alone.

**Comment by chrfritsch** [comment]
Here is the follow-up https://www.drupal.org/project/thunder/issues/3164391

**Review by dbosen — APPROVED** [review]

**Pull request merged**